cli/bin/grain.js

```diff
@@ -119,7 +119,7 @@ program
     "don't automatically import the Grain Pervasives module"
   )
   .graincOption("-o <filename>", "output filename")
-  .graincOption("--O0", "disable optimizations")
+  .graincOption("-O <level>", "set the optimization level")
```

Comment:
Should this default to level 3?

Reply:
It might be good to do that since it'd make the docs a bit nicer (you'd see the choices, and you'd see that 3 was the default). I was largely offloading the logic to `grainc` so we wouldn't have to keep them in sync. But it's probably better to do the better UX version.

Reply:
Yeah, better UX. This will eventually go away so we shouldn't worry about keeping it in sync for a little while.